### PR TITLE
fix(#16659): support className and style for message

### PR DESCRIPTION
**Pull request by kerm1it**
### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [x] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

[#16659](https://github.com/ant-design/ant-design/issues/16659) 

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |   Support customize Message with `className` and `style`       |
| 🇨🇳 Chinese |     全局消息支持自定义样式通过使用`className`和`style`     |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️ 

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

-----
[View rendered components/message/demo/custom-style.md](https://github.com/Kermit-Xuan/ant-design/blob/message-custom-style/components/message/demo/custom-style.md)
[View rendered components/message/index.en-US.md](https://github.com/Kermit-Xuan/ant-design/blob/message-custom-style/components/message/index.en-US.md)
[View rendered components/message/index.zh-CN.md](https://github.com/Kermit-Xuan/ant-design/blob/message-custom-style/components/message/index.zh-CN.md)

### CODE REVIEW

File: components/message/index.en-US.md [hunk 1]
```diff
@@ -16,23 +16,25 @@ Display global messages as feedback in response to user operations.
...
-- `message.info(content, [duration], onClose)`
-- `message.warning(content, [duration], onClose)`
-- `message.warn(content, [duration], onClose)` // alias of warning
-- `message.loading(content, [duration], onClose)`
+- `message.success(content, [duration], onClose, className, style)`
+- `message.error(content, [duration], onClose, className, style)`
+- `message.info(content, [duration], onClose, className, style)`
+- `message.warning(content, [duration], onClose, className, style)`
+- `message.warn(content, [duration], onClose, className, style)` // alias of warning
+- `message.loading(content, [duration], onClose, className, style)`
```
Comment (afc163) [inline components/message/index.en-US.md:24]: 别加到这里，加到 `message.xxx(config)` 里就行了。

**Comment by afc163** [comment]
feature branch

**Comment by afc163** [comment]
feature branch pls

**Comment by Kermit-Xuan** [comment]
> feature branch pls

这是什么意思？

**Comment by afc163** [comment]
https://ant.design/docs/react/contributing-cn#%E5%88%86%E6%94%AF%E7%AE%A1%E7%90%86

**Comment by Kermit-Xuan** [comment]
那我需要合并到feature上是吧？

**Comment by yoyo837** [comment]
> 那我需要合并到feature上是吧？

是

**Comment by yoyo837** [comment]
Move to #24024

**Pull request closed without merging**